libavformat/dashenc.c

```diff
@@ -1739,11 +1751,19 @@ static int dash_flush(AVFormatContext *s, int final, int stream)
             }
         }
 
-        if (!os->muxer_overhead)
-            os->muxer_overhead = ((int64_t) (range_length - os->total_pkt_size) *
+        if (!os->muxer_overhead) {
+            if (av_rescale_q(os->max_pts - os->start_pts, st->time_base, AV_TIME_BASE_Q) != 0)
```

Comment:
What is this 'if' mean here - what condition is needed to to rescale vs no rescale?

Reply:
You mean muxer_overhead, right? it is a parameter that is used to calculate bitrate.

Reply:
If you mean when does av_rescale_q() rescale, I should say, most of the time it rescales to original timabase (when timebase > 10000), and in the crash case it rescaled to zero (which was wrong). It should rescale to some big number, like 2000000 when timebase < 10000.